src/main.rs

```diff
@@ -1,29 +1,52 @@
 use std::process;
 use crossterm::style::{SetForegroundColor, Color};
+use config_manager::CustomColor;
 
 mod api;
 mod content_box;
+mod config_manager;
 
 /// Parses a string with predetermined colors
 /// ### Arguments
 ///
 /// * `key` - The key of the value
 /// * `text` - The content
+/// * `key_color` - The foreground color of the key
+/// * `text_color` - The foreground color of the content
 /// 
-/// ### Todo
-/// Make config file for colors in .config directory
-fn colorful_format(key: &str, text: String) -> String {
-  let main_color = SetForegroundColor(Color::Magenta);
-  let acccent_color = SetForegroundColor(Color::White);
+fn colorful_format(key: &str, text: String, key_color: Color, text_color: Color) -> String {
```

Comment:
i think text should pass by reference `text: &String` like the key so it doesn't get moved into here making it unaccessible outside

Reply:
Good point, I didn't think of that earlier